rc/filetype/c-family.kak

```diff
@@ -58,6 +59,11 @@ hook -group objc-highlight global WinSetOption filetype=objc %{
     hook -once -always window WinSetOption filetype=.* %{ remove-highlighter window/objc }
 }
 
+# Module aliases
+provide-module c %{ require-module c-family }
+provide-module cpp %{ require-module c-family }
+provide-module objc %{ require-module c-family }
+
```

Comment:
I think I'd rather have those *after* the module they reference, so that they dont get declared if declaring the c-family module fails for some reason.